moon-store-mac-os/moon-store-mac-os/Networking/Managers/UserManager.swift

```diff
@@ -110,4 +110,18 @@ final class UserManager {
             throw error
         }
     }
+    
+    // MARK: - Register user
+    
+    func registerUser(user: UserRegisterModel) async throws -> UserModel {
+        do {
+            let parameters = try networkManager.convertToDictionary(data: user)
```

Comment:
La convención de swift para el metodo ` convertToDictionary` seria `asDictionary(using: user)`

```suggestion
            let parameters = try networkManager.asDictionary(using: user)
```